RELEASE-LEADS.md

```diff
@@ -238,61 +154,92 @@ green light.
 - [knative-sandbox/eventing-kafka-broker](https://github.com/knative-sandbox/eventing-kafka-broker)
+- [knative-sandbox/eventing-natss](https://github.com/knative-sandbox/eventing-natss)
+- [knative-sandbox/eventing-prometheus](https://github.com/knative-sandbox/eventing-prometheus)
+- [knative-sandbox/eventing-rabbitmq](https://github.com/knative-sandbox/eventing-rabbitmq)
+- [knative-sandbox/sample-source](https://github.com/knative-sandbox/sample-source)
 
-The pins are similar to step 5 above, but now we're pinning `serving` and
-`eventing` respectively. Again, the pin PRs are sent against the **master**
-branch of each repository respectively.
+After both **eventing** and **serving**:
 
-### Cut `release-x.y` branches of all remaining repositories
+- [knative-sandbox/eventing-redis](https://github.com/knative-sandbox/eventing-redis)
+- [knative-sandbox/eventing-github](https://github.com/knative-sandbox/eventing-github)
+- [knative-sandbox/eventing-gitlab](https://github.com/knative-sandbox/eventing-gitlab)
 
-After the pin PRs are merged, cut the `release-x.y` branch in each of the
-remaining repositories (except `operator` and `client` as they are cut
-separately by the respective working group):
+Lastly:
 
-- [knative-sandbox/net-kourier](https://github.com/knative-sandbox/net-kourier)
-- [knative/eventing-contrib](https://github.com/knative/eventing-contrib)
-- [knative-sandbox/eventing-kafka-broker](https://github.com/knative-sandbox/eventing-kafka-broker)
+- [knative-sandbox/eventing-autoscaler-keda](https://github.com/knative-sandbox/eventing-autoscaler-keda)
+
+We have a few repos inside of Knative that are not handled in the standard
+process at the moment. They might have additional dependencies or depend on the
+releases existing. **Skip these**. Special cases are:
 
-Release automation will automatically pick up the branches and will likewise
-create the respective tags.
+- [knative/client](https://github.com/knative/client)
+- [knative/docs](https://github.com/knative/docs)
+- [knative/website](https://github.com/knative/website)
+- [knative/operator](https://github.com/knative/operator)
 
-## Right after the release
+## After the release
 
 Send a PR like [this one](https://github.com/knative/community/pull/209) to
 grant ACLs for the next release leads, and to remove yourself from the rotation.
 Include the next release leads in the PR as a reminder.
 
 ---
+
+## Cutting release branches.
+
+_Prerequisite_: Install the [**buoy** tool](https://github.com/test-infra).
+
+We have staged releasibility status checks that are sent to the release Slack
+channel to give alerts when things are ready to go. You can manually run that
+for each repo before cutting the release branch,
+
+```bash
+RELEASE=0.19
+if buoy check go.mod --domain knative.dev --release ${RELEASE} --verbose; then
+  git checkout -b release-${RELEASE}
+  ./hack/update-deps.sh --upgrade --release ${RELEASE}
+  git add .
+  git push origin release-${RELEASE}
```

Comment:
just to flag this up and make sure it's what we want: in the past version of the instructions this was done by PR, now we're pushing directly to the release branch?

Reply:
@mattmoor thoughts?